enhancements/insights/insights-operator-runtime.md

```diff
@@ -0,0 +1,532 @@
+---
+title: insights-operator-runtime
```

Comment:
Naming is difficult :), but is it a good name? I think I wouldn't use the word `operator`. Maybe something like `insights-node-runtime` or `insights-container-runtime` or `insights-runtime-extractor`